scene-recognition/process.py

```diff
@@ -1,9 +1,24 @@
-import pathlib
-import shutil
+# Process script to copy raw csv into new golds directory
+# 
+# - iterate through each raw directory (use yy-mm-dd pattern to find directories)
+# - copy each csv file to golds/timepoints directory (with some changes)
+# create csv files for TimeFrame annotations in golds/timeframes
+
 
+import re
+import shutil
+import csv
+from pathlib import Path
 import pandas as pd
 
 
+# Set this to True if you want the TimeFrames to be sensitive to sublabels, with a
+# sequence [S:D, S:H] you would get a TimeFrame of type S with the current setting,
+# but two TimeFrames with labels S:D and S:H if you include the sublabel. Given the 
+# way the current classifier is trained the former makes much more sense.
+USE_SUBLABEL = False
```

Comment:
I don't see point using this variable as well as how it's used in `_get_column_names`, `_labels_match`, `_create_timeframe` functions. Shouldn't we just have one and true gold format? We can just keep the "sub" label column in the gold format and leave which column to use to the users of the dataset, I guess.

Reply:
Yeah, it should not be where it was. I moved it out of the way into the deprecated function. I don't want to spend the time to figure out the exact uses for a deprecated function.